traffic/configure-ios.md

```diff
@@ -10,27 +10,33 @@ previous_url: /knowledge-base/configure-ios, get-started/mobile-traffic/configur
 
 # Capturing and Inspecting iOS Traffic
 
-This article describes how use Fiddler Everywhere to capture and inspect traffic that comes from iOS devices and simulators.
+This article describes how to use Fiddler Everywhere to capture and inspect traffic that comes from iOS devices and simulators.
 
-To capture and inspect traffic on iOS devices, perform the following steps:
+To capture and inspect traffic on iOS devices or emulators, perform the following steps:
 
 1. [Provide the prerequisites](#prerequisites).
 1. [Configure Fiddler Everywhere](#configuring-fiddler-everywhere).
-1. [Configure the iOS device](#configuring-the-ios-device).
+1. [Configure iOS device](#configure-ios-device).
+1. [Configure iOS emulator](#configure-ios-device).
```

Comment:
1. [Configure the iOS device](#configuring-the-ios-device).
1. [Configure the iOS emulator](#configuring-the-ios-emulator).